### PR TITLE
Add taxon for room and replace UI occurences with it

### PR DESCRIPTION
**Pull request by goran-peoski-work**
Resolves
- https://github.com/sparkletown/internal-sparkle-issues/issues/1098

A first (temporary) step towards a more comprehensive externalization of taxonomy and later UI facing text strings.
This PR replaces raw text with reference which will make it easy to list all occurrences and improve upon them in the future.

### CODE REVIEW

File: src/components/molecules/VenueRoomItem/VenueRoomItem.tsx [hunk 1]
```diff
@@ -110,15 +110,19 @@ export const VenueRoomItem: React.FC<VenueRoomItemProps> = ({
...
+            >
+              Add {ROOM_TAXON.lower}
             </Button>
           </Form>
         </Modal.Body>
       </Modal>
       <div className="VenueRoomItem" onClick={showModal}>
         <img
-          alt={`room-icon-${icon}`}
+          alt={`${ROOM_TAXON.lower} icon ${icon}`}
```
Comment (yurii-lubynets) [inline src/components/molecules/VenueRoomItem/VenueRoomItem.tsx:125]:
```suggestion
          alt={`${ROOM_TAXON.lower}-icon-${icon}`}
```

File: src/components/organisms/AdminVenueView/components/VenueRoomsEditor/VenueRoomsEditor.tsx [hunk 2]
```diff
@@ -287,7 +289,7 @@ export const VenueRoomsEditor: React.FC<VenueRoomsEditorProps> = ({
                 "Container__room-image--disabled": !room.isEnabled,
               })}
               src={room.image_url}
-              alt="room-logo"
+              alt={`${ROOM_TAXON.lower} logo`}
```
Comment (yurii-lubynets) [inline src/components/organisms/AdminVenueView/components/VenueRoomsEditor/VenueRoomsEditor.tsx:292]:
```suggestion
              alt={`${ROOM_TAXON.lower}-logo`}
```

**Review by yurii-lubynets — COMMENTED** [review]
LGTM. This solution helps us for the transition period.

One small ask: we have a different style for `alt` in `` sometimes it uses `text-text-text` style and in other places, there is a `text text text`. 

Please align `alt` usage in one style

**Comment by goran-peoski-work** [comment]
> LGTM. This solution helps us for the transition period.
> 
> One small ask: we have a different style for `alt` in `` sometimes it uses `text-text-text` style and in other places, there is a `text text text`.
> 
> Please align `alt` usage in one style


Reason behind it:

> Punctuation within alt attributes:
> As for any text, using punctuation in the text alternative makes the information easier to understand. In particular, remember to add space characters in the alt text when there’s no space character between the image and adjacent text, to avoid having words running together when they are read by a screen reader.

(from https://www.w3.org/WAI/tutorials/images/tips/ )

**Review by yurii-lubynets — APPROVED** [review]
LGTM

**Pull request merged**